### PR TITLE
Keycloak 15786

### PR DESCRIPTION
**Pull request by briandooley**

### CODE REVIEW

File: server_admin/topics/clients/con-advanced-settings.adoc [hunk 1]
```diff
@@ -1,72 +1,84 @@
 [id="con-advanced-settings_{context}"]
 ==== Advanced Settings
-There are 
+Advanced settings also available for OIDC Clients.
```
Comment (andymunro) [inline server_admin/topics/clients/con-advanced-settings.adoc]: How about: When you click *Advanced Settings*, additional fields appear.

File: server_admin/topics/clients/con-advanced-settings.adoc [hunk 2]
```diff
@@ -1,72 +1,84 @@
...
-
-If the following conditions are satisfied on a token request, {project_name} will bind an access token and a refresh token with a client certificate and issue them as holder-of-key tokens. If all conditions are not met, {project_name} rejects the token request.
+NOTE: To enable mutual TLS in {project_name}, see <<_enable-mtls-wildfly, Enable mutual SSL in WildFly>>.
 
-* The feature is turned on
-* A token request is sent to the token endpoint in an authorization code flow or a hybrid flow
-* On TLS handshake, {project_name} requests a client certificate and a client send its client certificate
-* On TLS handshake, {project_name} successfully verifies the client certificate
+Mutual TLS binds an access token and a refresh token together with a client certificate, which is exchanged during a TLS handshake. This prevents an attacker from using stolen tokens.
```
Comment (andymunro) [inline server_admin/topics/clients/con-advanced-settings.adoc]: should be .... "This binding prevents..."

File: server_admin/topics/clients/con-advanced-settings.adoc [hunk 3]
```diff
@@ -1,72 +1,84 @@
...
-*Proof Key for Code Exchange (PKCE)*
+* A token refresh request is sent to the token endpoint with a holder-of-key refresh token.
+* A UserInfo request is sent to UserInfo endpoint with a holder-of-key access token.
+* A logout request is sent to Logout endpoint with a holder-of-key refresh token.
 
-When an attacker steals an authorization code that was issued to a legitimate client, PKCE prevents the attacker from receiving the tokens that apply to that code.
+See https://tools.ietf.org/html/draft-ietf-oauth-mtls-08#section-3[Mutual TLS Client Certificate Bound Access Tokens] in the OAuth 2.0 Mutual TLS Client Authentication and Certificate Bound Access Tokens for more details.
 
-The administrator can select the following three options:
+WARNING: Currently, Keycloak client adapters do not support holder-of-key token verification. Keycloak adapters treat access and refresh tokens as bearer tokens.
```
Comment (andymunro) [inline server_admin/topics/clients/con-advanced-settings.adoc]: I would make this a NOTE.  Also "Keycloak" should be {project_name}.

File: server_admin/topics/clients/con-advanced-settings.adoc [hunk 4]
```diff
@@ -1,72 +1,84 @@
...
-There are 
+Advanced settings also available for OIDC Clients.
+
 [[_mtls-client-certificate-bound-tokens]]
 *OAuth 2.0 Mutual TLS Certificate Bound Access Tokens Enabled*
 
-Mutual TLS binds an access token and a refresh token with a client certificate exchanged during TLS handshake. This prevents an attacker who finds a way to steal these tokens from exercising the tokens. This type of token is called a holder-of-key token. Unlike bearer tokens, the recipient of a holder-of-key token can verify whether the sender of the token is legitimate.
-
-If the following conditions are satisfied on a token request, {project_name} will bind an access token and a refresh token with a client certificate and issue them as holder-of-key tokens. If all conditions are not met, {project_name} rejects the token request.
+NOTE: To enable mutual TLS in {project_name}, see <<_enable-mtls-wildfly, Enable mutual SSL in WildFly>>.
```
Comment (andymunro) [inline server_admin/topics/clients/con-advanced-settings.adoc]: Can NOTE work this way?  I thought it had to be
[NOTE] followed by 4 equals signs
content
4 equal signs.

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks so much better. Thanks.  A few comments only.

File: server_admin/topics/clients/con-advanced-settings.adoc (hunk 1 above)
Comment (briandooley) [reply server_admin/topics/clients/con-advanced-settings.adoc]: Going with "are displayed" as the style guide does not like "appear".

**Comment by briandooley** [comment]
Changes made based on feedback.

**Review by andymunro — APPROVED** [review]
Looks great. thanks.

**Comment by andymunro** [comment]
Thanks, Brian

**Pull request merged**